appveyor.yml

```diff
@@ -6,20 +6,6 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
```

Comment:
This just removes a confusing second license header.

Reply:
Yeah, just a little careless on my part not to check remnants from mycila generation.